db/helpers/seed_helper.rb

```diff
@@ -0,0 +1,29 @@
+class SeedHelper
+  def objectify(model)
+    return model.titleize.constantize unless model.include? "_"
```

Comment:
👍  This is not a problem, though, but I believe the next line alone can take care of both conditions.